extensions/wikia/Email/EmailMobileBadges.class.php

```diff
@@ -76,12 +76,12 @@ class EmailMobileBadges {
 	 * @return string - url to mobile badge
 	 */
 	public static function getBadgeFor( $language, $platform ) {
-		$badge = EmailMobileBadges::$badges[$language][$platform];
+		$images = EmailMobileBadges::$badges[$language];
```

Comment:
So this will still be a problem for anyone NOT using our top 10 languages.  At the strict warn level we have PHP configured for, the above will cause this type of warning for, say, Estonian users:
```
Notice: Undefined index: et in /usr/wikia/source/app/extensions/wikia/Email/EmailMobleBadges.class.php on line 79
```
An `empty()` check will take care of this.  Given `emtpy()` allows checking multiple keys at once, this is why I suggested making the check be `!empty( EmailMobileBadges::$badges[$language][$platform] )`

Basically, if you want to do any array access on something and you're not already fully confident your key is in there, you should do an `empty()` check.  Note that other methods like `array_key_exists()` and `isset()` also satisfy this but you'll see `empty()` used predominately through out MW code.